cmd/archive.go

```diff
@@ -55,6 +56,12 @@ func (c *cmdArchive) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if c.skipEnv {
+		c.gs.logger.Info("environment variables will be excluded from the archive")
```

Comment:
Considering it's something that manually has to be triggered by a CLI flag, I am not sure we need to noisily log it

```suggestion
		c.gs.logger.Debug("environment variables will be excluded from the archive")
```